test/other/test_em_js_side.c

```diff
@@ -14,9 +14,14 @@ EM_JS(void, js_side_func_void, (), {
   out(`hello from void func`);
 });
 
+EM_JS(void, js_side_func_indirect_ptr, (char **pptr), {
+  out(`indirect ptr value: ${pptr}`);
+});
+
 int test_side() {
   js_side_func(42, "hello");
   js_side_func2("hello again");
   js_side_func_void();
+  js_side_func_indirect_ptr(NULL);
```

Comment:
Can you pass `&"hello"` and then print the string js_side_func_indirect_ptr?

otherwise lgtm

Reply:
Yes you can use the makeGetValue macro.  See the libraries in src/lib for examples